Packs/NutanixHypervisor/Integrations/NutanixHypervisor/NutanixHypervisor.py

```diff
@@ -0,0 +1,1017 @@
+import copy
+from typing import Dict
+
+import pytz
+import urllib3
+
+from CommonServerPython import *
+
+# Disable insecure warnings
+urllib3.disable_warnings()
+
+''' CONSTANTS '''
+USECS_ENTRIES_MAPPING = {'boot_time_in_usecs': 'boot_time',
+                         'create_time_usecs': 'create_time',
+                         'start_time_usecs': 'start_time',
+                         'complete_time_usecs': 'complete_time',
+                         'last_updated_time_usecs': 'last_updated',
+                         'created_time_stamp_in_usecs': 'created_time',
+                         'last_occurrence_time_stamp_in_usecs': 'last_occurrence',
+                         'acknowledged_time_stamp_in_usecs': 'acknowledged_time',
+                         'resolved_time_stamp_in_usecs': 'resolved_time'}
+
+HOST_FIELDS_NOT_VERBOSE = [
+    "service_vmid",
+    "uuid",
+    "name",
+    "service_vmexternal_ip",
+    "hypervisor_key",
+    "hypervisor_address",
+    "hypervisor_username",
+    "controller_vm_backplane_ip",
+    "management_server_name",
+    "monitored",
+    "serial",
+    "state",
+    "vzone_name",
+    "cpu_model",
+    "num_cpu_cores",
+    "num_cpu_threads",
+    "num_cpu_sockets",
+    "hypervisor_full_name",
+    "hypervisor_type",
+    "num_vms",
+    "boot_time_in_usecs",
+    "is_degraded",
+    "is_secure_booted",
+    "is_hardware_virtualized",
+    "reboot_pending",
+    "cluster_uuid",
+    "has_csr",
+    "host_type",
+    "boot_time"
+]
+
+ALERT_FIELDS_NOT_VERBOSE = [
+    "id",
+    "alert_type_uuid",
+    "check_id",
+    "resolved",
+    "auto_resolved",
+    "acknowledged",
+    "service_vmid",
+    "node_uuid",
+    "created_time_stamp_in_usecs",
+    "last_occurrence_time_stamp_in_usecs",
+    "cluster_uuid",
+    "originating_cluster_uuid",
+    "severity",
+    "impact_types",
+    "classifications",
+    "acknowledged_by_username",
+    "message",
+    "detailed_message",
+    "alert_title",
+    "operation_type",
+    "acknowledged_time_stamp_in_usecs",
+    "resolved_time_stamp_in_usecs",
+    "resolved_by_username",
+    "user_defined",
+    "affected_entities",
+    "created_time",
+    "last_occurrence"
+]
+
+TIMEOUT_INTERVAL = 1
+
+UTC_TIMEZONE = pytz.timezone('utc')
+
+NUTANIX_HOST_FIELDS_TO_REMOVE = {'disk_hardware_configs', 'cpu_frequency_in_hz', 'cpu_capacity_in_hz',
+                                 'memory_capacity_in_bytes', 'stats', 'usage_stats'}
```

Comment:
```suggestion
HOST_FIELDS_TO_REMOVE = ['disk_hardware_configs', 'cpu_frequency_in_hz', 'cpu_capacity_in_hz',
                                 'memory_capacity_in_bytes', 'stats', 'usage_stats']
```

Reply:
@ShahafBenYakir Can i ask why you want me to transform the set to list? isn't it less efficient?

Reply:
Choose which structure you want but be consistent (you have all constants as lists now)